libs/features/mas/src/wcs.js

```diff
@@ -149,9 +149,9 @@ export function Wcs({ settings }) {
     }
 
     /**
-     * Flushes WCS cache
+     * Flushes cache
      */
-    function flushWcsCache() {
+    function flushCache() {
```

Comment:
commented in `mas-commerce-service` as well, this will confuse developers since these methods are copied on `mas-commerce-service` and there we don't know whether this method clears fragment cache or not.